src/graphics/post-effect.js

```diff
@@ -92,13 +92,7 @@ function createFullscreenQuad(device) {
 
     // Fill the vertex buffer
     const iterator = new VertexIterator(vertexBuffer);
-    iterator.element[SEMANTIC_POSITION].set(-1.0, -1.0);
-    iterator.next();
-    iterator.element[SEMANTIC_POSITION].set(1.0, -1.0);
-    iterator.next();
-    iterator.element[SEMANTIC_POSITION].set(-1.0, 1.0);
-    iterator.next();
-    iterator.element[SEMANTIC_POSITION].set(1.0, 1.0);
+    iterator.writeData(SEMANTIC_POSITION, [-1, -1, 1, -1, -1, 1, 1, 1], 4);
```

Comment:
Actually, you can do something like:

```
const data = new Float32Array([-1, -1, 1, -1, -1, 1, 1, 1]);
const vertexBuffer = new VertexBuffer(device, vertexFormat, 4, BUFFER_STATIC, data.buffer);
```

No need for an iterator.

Reply:
even better! done.